content/4-component-composition/1-props/angular-renaissance/userprofile.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, input } from "@angular/core";
+
+@Component({
+  standalone: true,
+  selector: "app-userprofile",
+  template: `
+    <p>My name is {{ name() }}!</p>
+    <p>My age is {{ age() }}!</p>
+    <p>My favourite colors are {{ favouriteColors().join(", ") }}!</p>
+    <p>I am {{ isAvailable() ? "available" : "not available" }}</p>
+  `,
+})
+export class UserprofileComponent {
```

Comment:
p in Profile should be uppercase :) User**p**rofileComponent

Reply:
I agree with you but my goal was to make the least changes as the current version with is also lowercase.
I guess it's not a big deal and I could indeed update it both here and in the older (current) version

Reply:
The primary goal is to show Angular best practices, so if it musts be uppercase in the two cases, let's change both naming.